compatibility/sandbox-migration/runner/Migration/Util.hs

```diff
@@ -0,0 +1,12 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Migration.Util (symDiff, equivalent) where
+
+import Data.List
+
+symDiff :: Eq a => [a] -> [a] -> [a]
+symDiff left right = (left \\ right) ++ (right \\ left)
+
+equivalent :: Eq a => [a] -> [a] -> Bool
```

Comment:
fwiw if you have an `Ord` instance which you can just derive you should be able to turn this into `equivalent xs ys = sort xs == sort ys` which has a more reasonable complexity.

Reply:
I know, I tried initially but adding `Ord` to the `deriving` clause on the relevant data types returned me an error that defied my knowledge of Haskell.

I just tried again and I probably just missed `deriving Ord` for `Party` as well. Thanks for raising this.

Reply:
As always, if you need help figuring out the error, I’m happy to help. Also happy to not change this as part of this PR.

Reply:
Address https://github.com/digital-asset/daml/pull/6376/commits/0de90238f5da1e55f38e42ea7b3aa0ea2662545f

Reply:
> As always, if you need help figuring out the error, I’m happy to help. Also happy to not change this as part of this PR.

Thanks, I just didn't realize that I didn't have all the necessary type classes instances in place, not it's solved. :bow: